WeTransferPRLinter/.gitignore

```diff
@@ -1,5 +1,4 @@
 .DS_Store
 /.build
 /Packages
-xcuserdata/
-Package.resolved
```

Comment:
Without having this checked in, we would always fetch dependencies, slowing down CI

Reply:
Yeah we should be committing the `.resolved` for sure, also for compatibility between ourselves.